README.md

```diff
@@ -1,4 +1,6 @@
+[![Build Status](https://travis-ci.org/Mifrill/railgun.svg?branch=master)](https://travis-ci.org/Mifrill/railgun)
 [![Quality](http://img.shields.io/codeclimate/github/zharikovpro/railgun.svg)](https://codeclimate.com/github/zharikovpro/railgun)
+[![Coverage Status](https://img.shields.io/codeclimate/coverage/github/Mifrill/railgun.svg)](https://codeclimate.com/github/mifrill/railgun)
```

Comment:
Coverage for this repo shouldn't display coverage from your repo. It should display coverage for zharikovpro/railgun repo, or skip it.